website/content/docs/commands/plugins/remove.mdx

```diff
@@ -35,8 +35,6 @@ Packer plugins by piping the results of `plugins installed` to `plugins removed`
 On a Unix based OS with xargs installed you can remove all plugin versions using the following command:
 ```shell-session
 ~>  packer plugins installed | xargs -n1 packer plugins remove
-/Users/dev/.config/packer/plugins/github.com/hashicorp/amazon/packer-plugin-amazon_v1.3.2_x5.0_darwin_arm64
-/Users/dev/.config/packer/plugins/github.com/hashicorp/azure/packer-plugin-azure_v2.1.3_x5.0_darwin_arm64
```

Comment:
I think we can leave those as example? Maybe settle with fake plugin names if amazon/azure being mentioned is a concern here.
I think it's good to keep this in just because people may not know about `xargs` (especially with options like `-n` or `-L` which are necessary here) and giving an example of the expected output may be a good indicator that they did the right thing.